src/main/java/pages/SettingsPage.java

```diff
@@ -42,15 +42,16 @@ public void changeSettingToRandomValue(String settingName) {
         openDropDown(settingName);
         String currentValue = getDropdownSelectedValue(settingName);
         dropdownOptions.stream().filter(webElement -> !webElement.getText().equals(currentValue))
```

Comment:
suggesting to replace 'equals' with 'equalsIgnoreCase'

Reply:
thx